src/app/offers/offers-card-large/offers-card-large.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="w-full h-full flex flex-col lg:flex-row justify-start shadow-md hover:shadow-xl
+            transition-all duration-300 rounded-3xl">
+  <div class="relative w-full lg:w-7/12">
+    <img src="{{ offer.titleImageSrc }}"
+         class="w-full h-full top-rounded-small-screen lg:rounded-l-3xl">
+    <div class="absolute float-left lg:top-6 left-6 lg:rounded-full
+    -bottom-10 lg:bottom-auto rounded-2xl
+    py-3 px-6 shadow-lg bg-white">
+      <div class="flex flex-col text-sm lg:flex-row">
+        <div>
+          <span class="font-bold lg:text-lg">{{ offer.roi }}</span>
+          <span class="ml-2 font-light">Annual interest rate</span>
+        </div>
+        <div class="mt-1 lg:mt-auto">
+          <span class="font-bold lg:text-lg lg:ml-8">{{ offer.minInvestment | currencyDefault }}</span>
+          <span class="ml-2 font-light">Minimum investment</span>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="w-full lg:w-5/12 bg-white bottom-rounded-small-screen md:rounded-r-3xl p-6">
+    <h1 class="mt-6 lg:mt-0 text-4xl uppercase tracking-wider font-light leading-relaxed">
+      {{ offer.title }}
+    </h1>
+    <p class="my-2 font-light text-gray-500 leading-relaxed">
+      {{ offer.shortDescription }}
+    </p>
+    <span class="mt-6 font-bold text-gray-800 text-sm">{{ offer.fundsRaised | currencyDefault }}</span>
+    <span class="mx-1 text-sm">of</span>
+    <span class="mt-6 font-bold text-gray-800 text-sm">{{ offer.fundsRequired | currencyDefault }}</span>
+    <div class="relative pt-1 mt-2">
+      <div class="overflow-hidden h-2 mb-2 text-xs flex rounded bg-gray-200">
+        <div style="width:30%"
+             class="shadow-none flex flex-col text-center whitespace-nowrap text-white justify-center bg-indigo-800"></div>
```

Comment:
why is whitespace-nowrap is needed here?

Reply:
When resizing screen, it has a tendency to break "My Portfolio" into two rows. Alternate solution was to create fixed width or min width, but this seems more general and elegant.